### DISCUSSION TITLE
What's meant by peer vars?

### DISCUSSION THREAD

**Discussion by mitya33** [tusen-ai/naive-ui#4907] [Q&A]
I've read the docs on [Customising Theme > Peer Vars](https://www.naiveui.com/en-US/os-theme/docs/customize-theme#Use-the-peers-vars) but I don't really understand what it's talking about. Would someone mind explaining in basic terms what this relates to? Peers as in what meaning? What is a peer in relation to what?

Many thanks.

**Comment by jahnli** [comment]
The scenario I understand is that, just as the scrollBar component in ` data-table` and `transfer` is a reusable `scrollBar`, the subject variables associated with it may not be exposed

**Reply by mitya33** [reply]
Thanks for your reply but I'm none the wiser. Perhaps it's just me, but the term "peer vars" doesn't have a generic meaning I'm familiar with and the docs associated with it explain nothing.
